chipy_org/apps/profiles/migrations/0004_rsvp_migration.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+import probablepeople
+
+
+def display_name_to_rsvps(apps, schema_editor):
+    UserProfile = apps.get_model('profiles', 'UserProfile')
```

Comment:
Why can't I use `apps.get_model('profiles', 'UserProfile')` in a meetings migration?